cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -519,6 +519,26 @@ func (ht *hcsTask) GetExec(eid string) (shimExec, error) {
 	return raw.(shimExec), nil
 }
 
+func (ht *hcsTask) GetExecs() (_ []shimExec, err error) {
+	var execs []shimExec
+	ht.execs.Range(func(key, value interface{}) bool {
+		wt, loaded := value.(shimExec)
+		if !loaded {
+			err = fmt.Errorf("failed to load exec %q", key)
+			return false
+		}
+		execs = append(execs, wt)
+		// Iterate all. Returning false stops the iteration. See:
+		// https://pkg.go.dev/sync#Map.Range
```

Comment:
I feel like this comment is no longer needed, especially since its in every `Range(` call

Reply:
Agree, removin!